src/views/MarketsDropdown.tsx

```diff
@@ -390,7 +390,9 @@ const $Popover = styled(Popover)`
   flex-direction: column;
 
   height: calc(
-    100vh - var(--page-header-height) - var(--market-info-row-height) - var(--page-footer-height)
+    100vh - var(--page-header-height) - var(--market-info-row-height) - var(--page-footer-height) - var(
+        --restriction-warning-height
```

Comment:
shouldnt this be --restriction-warning-currentHeight ?

Reply:
Markets dropdown is a desktop only component. Can change tho

Reply:
Yeah but you're setting the height of the banner and the banner shows up on mobile!

Reply:
This height calc doesn't set the height of the banner. It just takes the banner into the account for the MarketsDropdownContent.